packages/salesforcedx-vscode-apex/README.md

```diff
@@ -8,16 +8,16 @@ View outlines of Apex classes and triggers, see code-completion suggestions, and
 
 Before you set up this extension, make sure that you have these essentials.
 
-- **Java 8 Platform, Standard Edition Development Kit**  
-  Some features in Salesforce Extensions for VS Code depend upon the Java 8 Platform, Standard Edition Development Kit (JDK).  
+- **Java 8 Platform, Standard Edition Development Kit**
```

Comment:
Removing this whitespace removes the line break from this list item (at least in the GitHub web UI).

Reply:
Yes it does, thanks!
Which is a better fix, an extra line break or making that text a sub-bullet?

Reply:
Bulleted lists shouldn't have fewer than two items, so I'd prefer the line break.

Reply:
This one does actually two items.  How's this look?  
![image](https://user-images.githubusercontent.com/28264696/48452961-36fc8780-e766-11e8-8b2e-4fce0a1fe0d9.png)

Reply:
I ended up just using line breaks with the `<br/>` tag.